crates/notan_app/src/backend.rs

```diff
@@ -25,6 +25,9 @@ pub trait Backend: Downcast {
     /// Returns the window implementation
     fn window(&mut self) -> &mut dyn WindowBackend;
 
+    #[cfg(feature = "clipboard")]
+    fn set_clipboard_text(&mut self, text: &str);
```

Comment:
question: should this go in `WindowBackend`?

Reply:
I also wanted to add `get_clipboard_text`, but web's clipboard APIs are async and things got tricky real fast 🥲

Reply:
I don't have a strong opinion about if it should be on `WindowBackend` or not. It makes sense to me to have it there, but at the same time, we have `open_link` on the backend, and both (`open_link` and `set_clipboard_text`) depend on external crates (not winit), so yeah, not strong opinion. For me, it's ok how you made it, we can always move it later if we think it doesn't fit well there. 

About the `get_clipboard_text` on browsers, I tried to block the UI thread until the promise is resolved, and well... no surprises, it doesn't work at all. 

I guess we can go now without it, it's an edge case, but eventually, we should think of a way to do it. Maybe return an `Option<String>`, it always returns `Some("")` on desktop as default (even if it's nothing on the clipboard) and on web we return `None` while the promise is resolved and then `Some("")` when it's resolved. Just an idea.